pwa/src/templates/templateParts/cronjobs/CronjobDetailsTemplate.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import { Heading1 } from "@gemeente-denhaag/components-react";
+import { useTranslation } from "react-i18next";
+import { QueryClient } from "react-query";
+import { useCronjob } from "../../../hooks/cronjob";
+import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from "@gemeente-denhaag/table";
+
+interface CronjobDetailPageProps {
+  cronjobId: string;
+}
+
+export const CronjobsDetailTemplate: React.FC<CronjobDetailPageProps> = ({ cronjobId }) => {
+  const { t } = useTranslation();
+
+  const queryClient = new QueryClient();
+  const _useCronjob = useCronjob(queryClient);
+  const getCronjob = _useCronjob.getOne(cronjobId);
+
+  return (
+    <>
+      <Heading1>{t("Cronjob detail page")}</Heading1>
+
```

Comment:
I think it would be nice to have a back button here like so:
~~~~
      <div onClick={() => navigate("/cronjobs")}>
        <Link icon={<ArrowLeftIcon />} iconAlign="start">
          {t("Back to cronjobs")}
        </Link>
      </div>
~~~~